aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -394,6 +394,15 @@ impl AptosVM {
         )?)
     }
 
+    fn get_senders(txn_data: &TransactionMetadata) -> Vec<AccountAddress> {
```

Comment:
Shouldn't this check the feature flag and not pop if it's disabled even if GAS_PAYER_FLAG_BIT is set?

Reply:
We can never get here as the prologue would reject the tx due to inconsistent sequence number

Reply:
For the current code may be (need to double check) but this is potentially foot gunning if someone calls this as part of prologue